content/developer/rom/global/questionnaires.md

```diff
@@ -49,6 +51,19 @@ status: draft
       tot:  {label: "Somscore"},
       gedr: {label: "Problemen"},
       bep:  {label: "Dingen"}
+    },
+
+    sbg_info: {
+      key: "example_sbg_key",
+      questions: {
+        example_sbg_question_key_1: "example_1",
```

Comment:
Zullen we de documentatie al wel aanpassen om geen prefix te doen? 

En hierboven zeggen dat de use_legacy_keys in de toekomst default op false komt te staan.